japhethmutai/teamwork-backend
# Setup Test Framework and Travis CI

#### What does this PR do?
It sets up test framework using mocha and chai and integrates the project with Travis CI for continuous integration
#### Description of Task to be completed?
Create a test directory with a test file in the root directory
Create a test scenario and write a code to pass in app.js
Create a .travis.yml that integrates project with travis ci
#### How should this be manually tested?
Activate repo on travis CI and push the changes to github and observe code being tested on travis ci
#### Any background context you want to provide?
Mocha and chai should be installed and running
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
